lib/components/link.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Lilly.Components.Link do
+  use Surface.Component
+
+  prop uri, :uri, required: true
+
+  prop css_class, :css_class
+
+  slot default, required: false
+
+  def render(assigns) do
+    ~F"""
+    <a href={@uri}" class={"", @css_class}>
+      <#slot />
+    </a>
+    """
```

Comment:
We rarely use regular links in live view applications. What if we made this into a push_patch or push_redirect wrapper? Although that is built into surface.

Reply:
Great point. I'll revisit and make it more betterly.